BasicCoding/BasicCoding.xcodeproj/xcuserdata/miguel.askar.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We'll need to remove all this xcuserdata changes, they can be added to the .gitignore

Reply:
Fixed, gitignore fixed